it/src/main/java/com/google/cloud/teleport/it/common/JDBCBaseIT.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (C) 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.it.common;
+
+import com.google.cloud.teleport.it.TemplateTestBase;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import org.junit.Before;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class JDBCBaseIT extends TemplateTestBase {
+
+  private static final Logger LOG = LoggerFactory.getLogger(JDBCBaseIT.class);
+
+  // The sub-folder to store the jars in GCS.
+  private static final String GCS_PREFIX = "jars/";
+
+  // The .jar suffix for generating JDBC jar names
+  private static final String JAR_SUFFIX = ".jar";
+
+  // The JDBC Driver fully-qualified class names
+  protected static final String MYSQL_DRIVER = "com.mysql.jdbc.Driver";
+  protected static final String POSTGRES_DRIVER = "org.postgresql.Driver";
+  protected static final String ORACLE_DRIVER = "oracle.jdbc.driver.OracleDriver";
+  protected static final String MSSQL_DRIVER = "com.microsoft.sqlserver.jdbc.SQLServerDriver";
+
+  // The relative path to the JDBC drivers under Maven's `.m2/repository` directory
+  private static final String MYSQL_LOCAL_PATH = "mysql/mysql-connector-java";
+  private static final String POSTGRES_LOCAL_PATH = "org/postgresql/postgresql";
+  private static final String ORACLE_LOCAL_PATH = "com/oracle/ojdbc/ojdbc8";
+  private static final String MSSQL_LOCAL_PATH = "com/microsoft/sqlserver/mssql-jdbc";
+
+  // The name of the JDBC driver jars sans the .jar suffix
+  private static final String MYSQL_JAR_PREFIX = "mysql-connector-java";
+  private static final String POSTGRES_JAR_PREFIX = "postgresql";
+  private static final String ORACLE_JAR_PREFIX = "ojdbc8";
+  private static final String MSSQL_JAR_PREFIX = "mssql-jdbc";
+
+  // The versions of each of the JDBC drivers to use.
+  // NOTE: These versions must correspond to the versions declared in the `it/pom.xml` file.
```

Comment:
Can we crossref / mention in the pom that this class needs to be updated when we bump dependencies? This is an easy miss

Reply:
Done.